### PR TITLE
added ward_councillor_url to models

### PR DESCRIPTION
**Pull request by emre2038**

### CODE REVIEW

File: idp_data/idp_data/models.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Municipality(models.Model):
     name = models.CharField(max_length=200)
     homepage_url = models.CharField(max_length=500)
     by_laws_url = models.CharField(max_length=500)
+    ward_councillor_url = models.CharField(max_length=500,verbose_name="Find Your Ward Councillor Url")
```
Comment (paulmwatson) [inline idp_data/idp_data/models.py:19]: Small change but can we use `URL` and not `Url` for users, thanks.

File: idp_data/idp_data/migrations/0025_municipality_ward_councillor_url.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    dependencies = [
+        ('idp_data', '0024_municipality_by_laws_url'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='municipality',
+            name='ward_councillor_url',
+            field=models.CharField(default='', max_length=500, verbose_name='Find Your Ward Councillor Url'),
```
Comment (paulmwatson) [inline idp_data/idp_data/migrations/0025_municipality_ward_councillor_url.py:16]: Same here, `URL` not `Url` when displayed to users.

File: idp_data/idp_data/models.py (hunk 1 above)
Comment (emre2038) [reply idp_data/idp_data/models.py]: done 👍

File: idp_data/idp_data/migrations/0025_municipality_ward_councillor_url.py (hunk 2 above)
Comment (emre2038) [reply idp_data/idp_data/migrations/0025_municipality_ward_councillor_url.py:16]: created a new migration with `URL` keyword instead of `Url` or `url`

**Pull request merged**